test/functional/legacy/083_tag_search_with_file_encoding_spec.lua

```diff
@@ -0,0 +1,217 @@
+      'abc40	Xtags3.txt	/\x82`\x82a\x82b\n' ..
+      'abc41	Xtags3.txt	/\x82`\x82a\x82b\n' ..
+      'abc42	Xtags3.txt	/\x82`\x82a\x82b\n' ..
+      'abc43	Xtags3.txt	/\x82`\x82a\x82b\n' ..
+      'abc44	Xtags3.txt	/\x82`\x82a\x82b\n' ..
+      'abc45	Xtags3.txt	/\x82`\x82a\x82b\n' ..
+      'abc46	Xtags3.txt	/\x82`\x82a\x82b\n' ..
+      'abc47	Xtags3.txt	/\x82`\x82a\x82b\n' ..
+      'abc48	Xtags3.txt	/\x82`\x82a\x82b\n' ..
+      'abc49	Xtags3.txt	/\x82`\x82a\x82b\n' ..
+      'abc50	Xtags3.txt	/\x82`\x82a\x82b\n' ..
+      'abc51	Xtags3.txt	/\x82`\x82a\x82b\n' ..
+      'abc52	Xtags3.txt	/\x82`\x82a\x82b\n' ..
+      'abc53	Xtags3.txt	/\x82`\x82a\x82b\n' ..
+      'abc54	Xtags3.txt	/\x82`\x82a\x82b\n' ..
+      'abc55	Xtags3.txt	/\x82`\x82a\x82b\n' ..
+      'abc56	Xtags3.txt	/\x82`\x82a\x82b\n' ..
+      'abc57	Xtags3.txt	/\x82`\x82a\x82b\n' ..
+      'abc58	Xtags3.txt	/\x82`\x82a\x82b\n' ..
+      'abc59	Xtags3.txt	/\x82`\x82a\x82b\n' ..
+      'abc60	Xtags3.txt	/\x82`\x82a\x82b\n' ..
+      'abc61	Xtags3.txt	/\x82`\x82a\x82b\n' ..
+      'abc62	Xtags3.txt	/\x82`\x82a\x82b\n' ..
+      'abc63	Xtags3.txt	/\x82`\x82a\x82b\n' ..
+      'abc64	Xtags3.txt	/\x82`\x82a\x82b\n' ..
+      'abc65	Xtags3.txt	/\x82`\x82a\x82b\n' ..
+      'abc66	Xtags3.txt	/\x82`\x82a\x82b\n' ..
+      'abc67	Xtags3.txt	/\x82`\x82a\x82b\n' ..
+      'abc68	Xtags3.txt	/\x82`\x82a\x82b\n' ..
+      'abc69	Xtags3.txt	/\x82`\x82a\x82b\n' ..
+      'abc70	Xtags3.txt	/\x82`\x82a\x82b\n' ..
+      'abc71	Xtags3.txt	/\x82`\x82a\x82b\n' ..
+      'abc72	Xtags3.txt	/\x82`\x82a\x82b\n' ..
+      'abc73	Xtags3.txt	/\x82`\x82a\x82b\n' ..
+      'abc74	Xtags3.txt	/\x82`\x82a\x82b\n' ..
+      'abc75	Xtags3.txt	/\x82`\x82a\x82b\n' ..
+      'abc76	Xtags3.txt	/\x82`\x82a\x82b\n' ..
+      'abc77	Xtags3.txt	/\x82`\x82a\x82b\n' ..
+      'abc78	Xtags3.txt	/\x82`\x82a\x82b\n' ..
+      'abc79	Xtags3.txt	/\x82`\x82a\x82b\n' ..
+      'abc80	Xtags3.txt	/\x82`\x82a\x82b\n' ..
+      'abc81	Xtags3.txt	/\x82`\x82a\x82b\n' ..
+      'abc82	Xtags3.txt	/\x82`\x82a\x82b\n' ..
+      'abc83	Xtags3.txt	/\x82`\x82a\x82b\n' ..
+      'abc84	Xtags3.txt	/\x82`\x82a\x82b\n' ..
+      'abc85	Xtags3.txt	/\x82`\x82a\x82b\n' ..
+      'abc86	Xtags3.txt	/\x82`\x82a\x82b\n' ..
+      'abc87	Xtags3.txt	/\x82`\x82a\x82b\n' ..
+      'abc88	Xtags3.txt	/\x82`\x82a\x82b\n' ..
+      'abc89	Xtags3.txt	/\x82`\x82a\x82b\n' ..
+      'abc90	Xtags3.txt	/\x82`\x82a\x82b\n' ..
+      'abc91	Xtags3.txt	/\x82`\x82a\x82b\n' ..
+      'abc92	Xtags3.txt	/\x82`\x82a\x82b\n' ..
+      'abc93	Xtags3.txt	/\x82`\x82a\x82b\n' ..
+      'abc94	Xtags3.txt	/\x82`\x82a\x82b\n' ..
+      'abc95	Xtags3.txt	/\x82`\x82a\x82b\n' ..
+      'abc96	Xtags3.txt	/\x82`\x82a\x82b\n' ..
+      'abc97	Xtags3.txt	/\x82`\x82a\x82b\n' ..
+      'abc98	Xtags3.txt	/\x82`\x82a\x82b\n' ..
+      'abc99	Xtags3.txt	/\x82`\x82a\x82b\n' ..
+      'abc100	Xtags3.txt	/\x82`\x82a\x82b\n'
+      )
+
+    -- TODO what about the availability of iconv in neovim?
```

Comment:
Agree with @fwalch except instead of just `return`ing, call `pending()` so that the *total* test count stays the same, but we know which tests were skipped. E.g.:

```lua
if not has_iconv then
  pending('skipped (missing iconv)')
else
  it('is working', function()
  end)
end
```

See also: https://github.com/neovim/neovim/wiki/Unit-tests#guidelines-for-writing-tests